e2e/run.py

```diff
@@ -2,17 +2,128 @@
 
 import argparse
 import logging as l
+
 from pathlib import Path
 
 import e2e.channel as channel
 import e2e.client as client
 import e2e.connection as connection
 import e2e.packet as packet
+import e2e.relayer as relayer
 from e2e.cmd import Config
 from e2e.common import *
 
 
-def run(c: Config):
+def loop(c: Config):
+    IBC_0 = ChainId('ibc-0')
+    IBC_1 = ChainId('ibc-1')
+
+    TRANSFER = PortId('transfer')
+    CHANNEL = ChannelId('channel-0')
+
+    # 1. create some unreceived acks
+
+    # hermes tx raw ft-transfer ibc-0 ibc-1 transfer channel-0 10000 1000 -n 2
+    packet.packet_send(c, src=IBC_1, dst=IBC_0, src_port=TRANSFER,
+                       src_channel=CHANNEL, amount=10000, height_offset=1000, number_msgs=2)
+
+    # hermes tx raw ft-transfer ibc-1 ibc-0 transfer channel-0 10000 1000 -n 2
+    packet.packet_send(c, src=IBC_0, dst=IBC_1, src_port=TRANSFER,
+                       src_channel=CHANNEL, amount=10000, height_offset=1000, number_msgs=2)
+
+    sleep(5.0)
```

Comment:
Should we create a loop here which queries the chain to confirm that all the packets were sent and avoid the race condition here?